avocado/core/output.py

```diff
@@ -608,19 +608,23 @@ class LoggingFile(object):
     """
 
     def __init__(self, prefix='', level=logging.DEBUG,
-                 logger=[logging.getLogger()]):
+                 loggers=None):
```

Comment:
As a matter of fact, this fixes another problem, the fact we're passing a mutable value as the default for the loggers parameter. Nice.